.gitattributes

```diff
@@ -1,7 +1,9 @@
 # Ignore dev/testing files for distribution builds
-/tests export-ignore
+/test export-ignore
```

Comment:
Does `/tests` have a different effect from `/test`?

Reply:
I believe so as `/tests` is a different path than `/test`. As this change was made by Pavel, I trust him 😂  But I think the idea here is that previously the repo had `tests` folder which is deleted in this new version, and now we have a new folder named `test`.